packages/valory/skills/abstract_round_abci/skill.yaml

```diff
@@ -29,12 +29,61 @@ fingerprint:
   test_tools/common.py: bafybeigzrzrww6swkcbfqd34apgnrmcbgmqjjwikfj6b7jcayuwoqhnyve
   test_tools/integration.py: bafybeihnvdcemxv2y7w7rokr5atmmzfy7jtwc54qj43paanzir2ppfvsby
   test_tools/rounds.py: bafybeignmwjc2qpia6lz6qtdj3lmvrf7fw64oho5iwxlp5hdez5o3nvv3a
+  tests/.hypothesis/examples/041a0496c3275b08/6f17d6ad84cf1099: bafybeicmiephpndjuutph6jsow5fggb5jlyc7smb7ikdcxityga7zgwoeq
+  tests/.hypothesis/examples/0b4f77d8d2028b59/22d6e0043782e008: bafybeiaztz3okq2jfdvgtdnmzs3cdfeiuxgkdmphfhbfbi2xyv6yfupf7q
+  tests/.hypothesis/examples/0b4f77d8d2028b59/2d360c42c2636244: bafybeib42ufeocjkbokfoaob6h57b7qjkdztidrn25lkwblwgdep5uzs2m
+  tests/.hypothesis/examples/0b4f77d8d2028b59/4ce185fb1be22dfb: bafybeibacddhjmjpt5qbzhux634avms4stmvmvomv7yukk5zjkyii7zzz4
```

Comment:
Will fix this. May I suggest that we open an issue to `rm -r` all the `.hypothesis` before running the generators?

Reply:
Opened #1374.